sections/ansible.zsh

```diff
@@ -0,0 +1,50 @@
+#
+# Ansible
+#
+# Ansible is an IT automation tool. It can configure systems, deploy software, and orchestrate more advanced IT tasks
+# such as continuous deployments or zero downtime rolling updates.
+# Link: http://docs.ansible.com/ansible/latest/index.html
```

Comment:
https://docs.ansible.com/ansible/latest/index.html

Reply:
I think it's worth stressing: `http`**S**. Had to scan-through the link a few times to notice the difference :)

Reply:
```suggestion
# Link: https://docs.ansible.com/ansible/latest/index.html
```

Reply:
done